config/ccg_bert.conf

```diff
@@ -0,0 +1,33 @@
+include "defaults.conf"	
```

Comment:
If this is meant to be a useful examples, add at least a few comments. (Follow demo.conf or the new copa_bert.conf.)

Reply:
This isn't going to contribute anything that the COPA example doesn't so I don't think this will be an example (it would be more for people who are running CCG, and also to address the issue opened about the config file).

Reply:
Sounds good. Just add a one-line header explaining what it's for, at least?

Reply:
Just a one-liner saying what this is?